pkg/sidecar/config/config.go

```diff
@@ -226,7 +239,7 @@ func (s *ScyllaConfig) setupEntrypoint(ctx context.Context) (*exec.Cmd, error) {
 	scyllaCmd := exec.Command(entrypointPath, args...)
 	scyllaCmd.Stderr = os.Stderr
 	scyllaCmd.Stdout = os.Stdout
-	s.logger.Info(ctx, "Scylla entrypoint command:\n %v", scyllaCmd)
+	s.logger.Info(ctx, "Scylla entrypoint", "command", scyllaCmd)
```

Comment:
Nice fix. I didn't know that loggers aren't supposed to have newlines back then.

Reply:
The new logger is adamant about even number of arguments. Before this it actually panicked which is for sure something we want to avoid.

Reply:
Huh I should take a better look at the new logger then.